application-workloads/sharepoint/sharepoint-adfs/dsc/ConfigureFEVM.ps1

```diff
@@ -561,6 +573,32 @@ configuration ConfigureFEVM
             PsDscRunAsCredential = $DomainAdminCredsQualified
             DependsOn            = "[CertReq]SPSSiteCert", "[SPFarm]JoinSPFarm"
         }
+
+        if ($EnableAnalysis) {
+            # This resource is for analysis of dsc logs only and totally optionnal
+            xScript parseDscLogs
+            {
+                TestScript = { return $false }
+                SetScript = {
+                    $setupPath = $using:SetupPath
+                    $localScriptPath = "$setupPath\parse-dsc-logs.py"
+                    New-Item -ItemType Directory -Force -Path $setupPath
+
+                    $url = "https://gist.githubusercontent.com/Yvand/777a2e97c5d07198b926d7bb4f12ab04/raw/parse-dsc-logs.py"
```

Comment:
this needs to be included with the sample...

Reply:
Understood, I just commented that portion of code to be compliant.
I use it only during development to analyze timings and make optimizations. Here it is de facto disabled because it depends on variable enableAnalysis, which I hardcoded to false.